packages/config-loader/src/lib/schema/types.ts

```diff
@@ -96,6 +96,11 @@ type ConfigProcessingOptions = {
    * will be omitted.
    */
   valueTransform?: TransformFunc<any>;
+
+  /**
+   * Whether or not to include the `filteredKeys` property in the output `AppConfig`s.
```

Comment:
Maybe mention what the default is

Reply:
Sure, would say that it's pretty universal that optional booleans default to false tho 😅